components/atoms/SortSelect.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState, ChangeEvent } from "react";
+import MenuItem from "@material-ui/core/MenuItem";
+import MuiSelect from "@material-ui/core/Select";
+import useSelectStyles from "styles/select";
+import useInputStyles from "styles/input";
+
+const options = [
+  {
+    value: "name-descend",
+    label: "名前順（A-Z）",
+  },
+  {
+    value: "name-ascend",
+    label: "名前順（Z-A）",
+  },
+  {
+    value: "created-at-descend",
+    label: "作成日順（新しい）",
+  },
+  {
+    value: "created-at-ascend",
+    label: "作成日順（古い）",
+  },
+  {
+    value: "updated-at-descend",
+    label: "更新日順（新しい）",
+  },
+  {
+    value: "updated-at-ascend",
+    label: "更新日順（古い）",
+  },
+];
```

Comment:
```suggestion
] as const;
```

理由:
このような型になると思うのですが

```ts
const options: {
    value: string;
    label: string;
}[]
```

[const assertions](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions)を使うことでより厳格になります。

```ts
const options: readonly [{
    readonly value: "name-descend";
    readonly label: "名前順（A-Z）";
}, {
    readonly value: "name-ascend";
    readonly label: "名前順（Z-A）";
}, {
    readonly value: "created-at-descend";
    readonly label: "作成日順（新しい）";
}, {
    ...;
}, {
    ...;
}, {
    ...;
}]
```

より厳格になることで意図せぬ変更に強くなって安全かなと感じました。あまり重要じゃない部分かもですが一応ご検討ください。

Reply:
便利そうですね！ありがとうございます！